stix_shifter/stix_transmission/src/modules/security_advisor/security_advisor_connector.py

```diff
@@ -0,0 +1,26 @@
+from ..base.base_connector import BaseConnector
+from .security_advisor_ping import SecurityAdvisorPing
+from .security_advisor_query_connector import SecurityAdvisorQueryConnector
+from .security_advisor_status_connector import SecurityAdvisorStatusConnector
+from .security_advisor_results_connector import SecurityAdvisorResultsConnector
+from .security_advisor_delete_connector import SecurityAdvisorDeleteConnector
+from .security_advisor_auth import SecurityAdvisorAuth
+
+class Connector(BaseConnector):
+    def __init__(self, connection, configuration ):
+        
+        auth = {}
+        accountID = configuration.get("accountID")
+        apiKey = configuration.get("apiKey") 
+        host = connection.get("host")
+
+        authToken = SecurityAdvisorAuth(apiKey)
+        auth["accountID"] = accountID
+        auth["authToken"] = authToken
```

Comment:
The configuration already has an auth key when it gets passed in, you don't need to create a new one here.